### PR TITLE
[HTP-0001] NoBid IX Adapter: Initial pull request

### PR DESCRIPTION
**Pull request by redaguermas**
## Type of Change

- [ X ] New adapter
- [ ] Feature Update
- [ ] Bug Fix
- [ ] Code Refactor
- [ ] Other

## Description of Change
This code in the Adapter is not complete yet. I am getting errors in the build. I want to pass this phase first.

### CODE REVIEW

**Comment by redaguermas** [comment]
Any update on this PR?
Thanks

**Comment by jas-singh** [comment]
hi @redaguermas 

Please contact adapter-certifications@indexexchange.com or khushi.irani@indexexchange.com to get an update on this PR.

**Comment by ix-certification** [comment]
Hi @redaguermas 
Testing with IE 11, found that a test is failing. 
The targeting is not being set properly.
image

File: no-bid/no-bid-htb-system-tests.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    return 'NONE';
+}
+
+function getArchitecture() {
+    return 'SRA';
+}
+
+function getConfig() {
+    return {
+        siteId: {
```
Comment (ix-certification) [inline no-bid/no-bid-htb-system-tests.js:21]: The validator is expecting `siteId` to be a number. However, here we are passing in an object. Can you update the config to reflect the correct value ?

Comment (redaguermas) [reply no-bid/no-bid-htb-system-tests.js]: I changed the getConfig() function to return siteId as a number instead of an object.

Comment (redaguermas) [reply no-bid/no-bid-htb-system-tests.js]: @ix-certification Hello.
Thank you for looking into the root cause of the error. Can you please check the change we made yesterday?

-Regards

Comment (ix-certification) [reply no-bid/no-bid-htb-system-tests.js]: Hello
No problem. The changes look good.

**Comment by redaguermas** [comment]
Hello @ix-certification,
Thank you for looking into the issue and fixed the test. Can you please review and let us know.

Regards

**Comment by redaguermas** [comment]
Hello @ix-certification are the tests still failing on IE11?
Any updates on this?

**Comment by redaguermas** [comment]
@ix-certification Sorry I did not see your reply on the thread. Thank you for reviewing the change.
When can we expect this branch to be merged?

**Comment by ix-certification** [comment]
Hi @redaguermas 
We will merge your changes shortly .

**Pull request merged**